enterprise/server/tasksize/tasksize.go

```diff
@@ -178,7 +182,10 @@ func (s *taskSizer) Update(ctx context.Context, cmd *repb.Command, md *repb.Exec
 	}
 	statusLabel := "ok"
 	defer func() {
-		props := platform.ParseProperties(&repb.ExecutionTask{Command: cmd})
+		props, err := platform.ParseProperties(&repb.ExecutionTask{Command: cmd})
+		if err != nil {
+			log.CtxWarningf(ctx, "Failed to parse task properties: %s", err)
+		}
```

Comment:
This will mostly be a no-op, but adding error handling here just to future proof